{
  "researchdata": [
    {
      "created_at": "2020-12-13T06:31:45",
      "updated_at": "2020-12-13T06:31:45",
      "html_url": "https://github.com/pulibrary/researchdata/pull/69",
      "number": 69,
      "pull_request_html_url": "https://github.com/pulibrary/researchdata/pull/69",
      "repository_html_url": "https://github.com/pulibrary/researchdata",
      "repository_name": "researchdata",
      "state": "open",
      "title": "Bump ini from 1.3.5 to 1.3.8 in /core",
      "user_name": "dependabot[bot]",
      "comments": [],
      "events": [
        {
          "actor_name": null,
          "created_at": "2020-12-13T06:31:46",
          "type": "labeled"
        },
        {
          "actor_name": null,
          "created_at": "2020-12-13T06:31:46",
          "type": "labeled"
        }
      ],
      "pr_html_url": "https://github.com/pulibrary/researchdata/pull/69",
      "action": "pull_request_open"
    }
  ],
  "uptime": [
    {
      "created_at": "2020-12-01T03:22:45",
      "updated_at": "2020-12-01T03:22:45",
      "html_url": "https://github.com/pulibrary/uptime/issues/19",
      "number": 19,
      "pull_request_html_url": null,
      "repository_html_url": "https://github.com/pulibrary/uptime",
      "repository_name": "uptime",
      "state": "closed",
      "title": "ðŸ›‘ Confluence is down",
      "user_name": "Francis Kayiwa",
      "comments": [
        {
          "user_name": "Francis Kayiwa",
          "updated_at": "2020-12-13T08:55:18",
          "html_url": "https://github.com/pulibrary/uptime/issues/19#issuecomment-743975518",
          "body": "**Resolved:** Dataspace is back up in [`07618c6`](https://github.com/pulibrary/uptime/commit/07618c6bfda4d1f1b25b404133ebba79d9ab9406\n)."
        }
      ],
      "events": [],
      "pr_html_url": null,
      "action": "closed"
    }
  ],
  "__meta__": {
    "issue_count": 2,
    "closed_count": 1,
    "pull_request_open_count": 1,
    "today": "2020-12-13T22:00:03",
    "yesterday": "2020-12-12T22:00:03"
  }
}